src/templates/dillo/user.pug

```diff
@@ -5,7 +5,7 @@
 
 #col_main.listing-list
 	#list-hits
-		| {% for post in posts['_items'] %}
+		| {% for post in posts %}
```

Comment:
If a user has 100s of posts we should be careful and have pagination (this is for later).